indico/web/client/js/react/components/ResponsivePopup.jsx

```diff
@@ -0,0 +1,17 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import {Popup} from 'semantic-ui-react';
+import {useResponsive} from 'indico/react/util';
+
+const ResponsivePopup = props => {
```

Comment:
`export default function ResponsivePopup(props) { ...`

Reply:
Is this because of the naming problem when importing it?

Reply:
i think in this particular case the name would be derived correctly, but generally yes (and CRA etc. all use `function` for their top-level functions representing components)